CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
+    {
+        // GET api/<controller>
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/<controller>/5
+        public string Get(int id)
+        {
+            return "value1";
+        }
+
+        // POST api/<controller>
+        public void Post([FromBody]string value)
+        {
+        }
+
+        // PUT api/<controller>/5
+        public void Put(int id, [FromBody]string value)
```

Comment:
It's only template code.

I didn't want to put much effort into it, as it's likely to get deleted later anyway.

It was added to help demonstrate the Swagger interface (otherwise there would be nothing to show).